cddl/protected-corim-header-map.cddl

```diff
@@ -1,6 +1,16 @@
-protected-corim-header-map = {
-  &(alg: 1) => int
-  &(content-type: 3) => "application/rim+cbor"
-  &(corim-meta: 8) => bstr .cbor corim-meta-map
-  * cose-label => cose-value
-}
+; protected header map MUST contain at least one of corim-meta (8) or CWT-Claims (15)
```

Comment:
>MUST

Are comments in CDDL normative?

Reply:
I will add that to my "normative language" scrubs I have been doing on the rest of the document.